hyper/http11/connection.py

```diff
@@ -192,6 +204,23 @@ def request(self, method, url, body=None, headers=None):
 
         return
 
+    def _headers_to_http_header_map(self, headers):
+        headers = headers or {}
+        if not isinstance(headers, HTTPHeaderMap):
+            if isinstance(headers, Mapping):
+                headers = HTTPHeaderMap(headers.items())
+            elif isinstance(headers, Iterable):
+                headers = HTTPHeaderMap(headers)
+            else:
+                raise ValueError(
+                    'Header argument must be a dictionary or an iterable'
+                )
+        return headers
```

Comment:
This "method" does not use the word `self` anywhere inside it, and so is not a method at all: it's a function pretending to be one. ;)